app/jobs/v3/buildpack_cache_delete.rb

```diff
@@ -2,10 +2,12 @@ module VCAP::CloudController
   module Jobs
     module V3
       class BuildpackCacheDelete < VCAP::CloudController::Jobs::CCJob
-        attr_accessor :app_guid
+        attr_accessor :app_guid, :resource_guid, :resource_type, :model_class
 
-        def initialize(app_guid)
+        def initialize(model_class, app_guid, resource_type=nil)
+          @model_class = model_class
```

Comment:
What are `model_class` and `resource_type` used for here?

Reply:
Indeed, model_class is not necessary. resource_guid is expected by pollable_job_wrapper. But then we don't need app_guid. Will change app_guid to resource_guid.